### PR TITLE
Fix server crashes

### LINKED ISSUE #69: Server crashes
**Describe the bug**
Able to crash the server by sending a query request

**To Reproduce**
1. Go to https://api-tournamanager.herokuapp.com/graphiql
2. Send a mutation request = mutation{ createTeam(name: "Team A") }
3. Send a query request = query{ teams{id name }}

**Expected behavior**
Should have returned the added Team as  JSON response

**Screenshots**
afbeelding

**Desktop Smartphone:
 - Irrelevant

**Additional context**
Afterwards server did not respond to any requests.

### PR DESCRIPTION
**Pull request by MindHawk**
Pull request fixes #69

### CODE REVIEW

File: src/main/java/com/fontys/api/queries/TeamQuery.java [hunk 1]
```diff
@@ -18,7 +18,7 @@ public TeamQuery(TeamService teamService) {
         this.teamService = teamService;
     }
 
-    public List<Team> teams() {
+    public List<Team> teams(int count) {
```
Comment (lfb0801) [inline src/main/java/com/fontys/api/queries/TeamQuery.java]: can this be made a nullable parameter or with a default value?

**Review by lfb0801 — CHANGES_REQUESTED** [review]

File: src/main/java/com/fontys/api/queries/TeamQuery.java (hunk 1 above)
Comment (MindHawk) [reply src/main/java/com/fontys/api/queries/TeamQuery.java]: On it.

Comment (MindHawk) [reply src/main/java/com/fontys/api/queries/TeamQuery.java]: Thought Java does not support nullable ints. After some research by @lfb0801 , turns out you can use the integer **class** and use the @Nullable annotation.
Now made nullable.

**Review by lfb0801 — APPROVED** [review]

**Review by Zeehond99 — APPROVED** [review]
Good job.

**Review by chf3kl9 — APPROVED** [review]

**Review by larsvandriel — APPROVED** [review]

**Review by MichielLangen — APPROVED** [review]
***nice***

**Review by Zeehond99 — APPROVED** [review]

**Review by lmtjanssen — APPROVED** [review]

**Pull request merged**